File: tests/11_slli_1.py
#!/usr/bin/env python3

from interp import *

def t():
    c = [
    ]
    i = [
        cpu_encode_op_li_i64(0,1),
        cpu_encode_op_slli_i64(0,16),
        cpu_encode_op_break(0),
    ]
    cpu_test("slli_1", c, i)

t()

File: tests/11_slli_1.go
package main

import . "glyph/tests/interp"

func t() {
    c := []uint64{
    }
    i := []uint16{
        CPU_encode_op_movi_i64(0,1),
        CPU_encode_op_slli_i64(0,16),
        CPU_encode_op_break(0),
    }
    CPU_test("slli_1", c, i);
}

func main() {
    t()
}
